tests/components/opentherm_gw/conftest.py

```diff
@@ -39,3 +45,18 @@ def mock_pyotgw() -> Generator[MagicMock]:
         ),
     ):
         yield mock_gateway
+
+
+@pytest.fixture
+def mock_config_entry() -> MockConfigEntry:
```

Comment:
Do we have a unique_id?

Reply:
It does not seem to be mandatory, but I can add it anyway

Reply:
Let's add it for completeness